README.md

```diff
@@ -120,24 +120,22 @@ Example:
 
 ## 👥 Authors <a name="authors"></a>
 
-
+1. Hien Phan
 - GitHub: [@hienphan0111](https://github.com/hienphan0111)
 - Twitter: [@hienphan0111](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/hien-phan-61097b256/)
-
+2. Nezerwa
+- GitHub: [@Nezerwa](https://github.com/Nezerwa)
```

Comment:
- [optional] It will be more professional to add more links, most especially Linkedln. Thanks. 👍